### PR TITLE
Fix invalid datetime value in test_issue_8

### PR DESCRIPTION
**Pull request by d-ryzhikov**
## What do these changes do?

Fix unix socket connection and datetime value in tests.

## Are there changes in behavior for the user?

No behavior changes.

## Related issue number

https://github.com/aio-libs/aiomysql/issues/338

## Checklist

- [x] I think the code is well written
- [x] ~~Unit tests for the changes exist~~
- [x] ~~Documentation reflects the changes~~
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` (e.g. `588.bugfix`)
  * if you don't have an `issue_id` change it to the pr id after creating the PR
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: `Fix issue with non-ascii contents in doctest text files.`

### CODE REVIEW

File: tests/test_issues.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def test_issue_8(self):
         yield from c.execute("drop table if exists test")
         yield from c.execute("""CREATE TABLE `test` (`station` int(10) NOT
             NULL DEFAULT '0', `dh`
-            datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
+            datetime NOT NULL DEFAULT '1000-01-01 00:00:00',
```
Comment (d-ryzhikov) [inline tests/test_issues.py]: According to [MySQL docs](https://dev.mysql.com/doc/refman/5.7/en/datetime.html), supported range for dates is `1000-01-01` to `9999-12-31`.

File: tests/fixtures/my.cnf.tmpl [hunk 2]
```diff
@@ -7,12 +7,12 @@ port = {port}
 host = {host}
 password = {password}
 database = {db}
-socket = /var/run/mysqld/mysqld.sock
+socket = /tmp/aiomysql/mysqld.sock
```
Comment (d-ryzhikov) [inline tests/fixtures/my.cnf.tmpl]: This socket path and the following ones are taken from [docker-compose volume config](https://github.com/aio-libs/aiomysql/blob/master/docker-compose.yml#L11).

**Comment by d-ryzhikov** [comment]
No sure how to fill in changes for this PR.

**Comment by Nothing4You** [comment]
this was already changed in 8a016be6516a09cdc65d7718783e5ea089a89dd1

**Pull request closed without merging**